### DISCUSSION TITLE
Get python program output

### DISCUSSION THREAD

**Discussion by sregister** [P403n1x87/austin#169] [Q&A]
Hi

When profiling my program with austin from the CLI I am unable to see the terminal output generated by my program while it is running. Is there anyway to get this back? The program I'm profiling isn't 100% deterministic and I want to make sure I'm profiling the desired flow so getting output from the python program itself (ideally in realtime) would be awesome.

**Comment by P403n1x87** [comment]
Hi @sregister. You can get the original output back if you use the `-o/--output` option to save the collected samples to a file.
